### ISSUE TITLE
[Quest] Ruuna the Blind - available right away

### ISSUE THREAD

**Issue by Greywolf59** [WoWManiaUK/Redemption#6120]
Quest [Ruuna the Blind](https://www.wowhead.com/wotlk/quest=12425/ruuna-the-blind) is available to player right away, without any prerequisites.
However the text of the quest states:

"This trapper business has the potential to be more important than we've accounted for.

If they're not with the [Alliance](https://wowpedia.fandom.com/wiki/Alliance), there's a good chance they're with the [Scourge](https://wowpedia.fandom.com/wiki/Scourge), which could mean trouble.

Our previous efforts might've paid off, though. We spotted our former captive meeting with a fortune teller of some sort before we lost track of her.

I'm almost certain now that she will take us to their leader if we can find her again. Find [Ruuna the Blind](https://wowpedia.fandom.com/wiki/Ruuna_the_Blind) and see what you can find out."

This suggests this quest should be available only after player does quests concerning Silverbrook.

Wowhead comment: https://www.wowhead.com/wotlk/quest=12425/ruuna-the-blind#comments:id=465446
https://wowpedia.fandom.com/wiki/Mikhail%27s_Journal_(quest)#Progress

**Comment by Greywolf59** [comment]
When I came to [Ruuna](https://www.wowhead.com/wotlk/npc=27581/ruuna-the-blind) to hand in the quest [Ruuna the Blind](https://www.wowhead.com/wotlk/quest=12425/ruuna-the-blind), she had quest [Ruuna's Request](https://www.wowhead.com/wotlk/quest=12328/ruunas-request) already available, before I handed in the quest. So it seems this quest is also missing some prerequisites, as to how much earlier it is available I can't say.

EDIT: moved to separate issue.

**Comment by Echinaceax** [comment]
![image](https://github.com/WoWManiaUK/Redemption/assets/39439201/1d58a536-79e6-4d72-8f04-89ad5fcb8bef)

need to see where tho, here maybe https://www.wowhead.com/wotlk/quest=12164/hour-of-the-worg

edited info
after clicking previous quest a few times it found some info and traced it back to here  https://wowwiki-archive.fandom.com/wiki/Quest:Out_of_Body_Experience


Says starts here https://www.wowhead.com/wotlk/quest=12164/hour-of-the-worg#comments:id=465417

![image](https://github.com/WoWManiaUK/Redemption/assets/39439201/e7d16f1e-8a9b-4229-afa8-56d16623c8c1)

@Greywolf59 
need these fixes in seperate reports or one full questline report for Grizzly Hills (cant patch fix like this)

![image](https://github.com/WoWManiaUK/Redemption/assets/39439201/07a3cc1a-98a1-4627-b107-2bffe9629df4)


quite mixed info online atm...

**Comment by Echinaceax** [comment]
Maine checked info for this on the other report, we can fix all in https://github.com/WoWManiaUK/Redemption/issues/6121 so i will close this one

thx @Greywolf59
